Marlin/src/pins/pins_BIGTREE_SKR_V1.3.h

```diff
@@ -212,11 +212,17 @@
 
 #endif // ULTRA_LCD
 
+#if ENABLED(MKS_MINI_12864)       //assign pins for MKS_MINI_12864
+	#define DOGLCD_CS P1_21
+	#define DOGLCD_A0 P1_22
+	#define USB_SD_ONBOARD        // Provide the onboard SD card to the host as a USB mass storage device
+	#define LPC_SD_LCD            // MKS_MINI_12864 uses SD card on the display
+#else
```

Comment:
Tabs are verboten! Please configure your editor to use 2 spaces instead of tab characters.